tasks/preload.js

```diff
@@ -202,21 +202,35 @@ module.exports = function (grunt) {
 						
 						var iOriginalSize, iCompressedSize;
 
+						var parameterName;
+
+						// Default options for UglifyJS
+						var uglifyParameters = {
+							fromString: true,
+							warnings: grunt.option('verbose') === true,
+							output: {
+								comments: copyrightCommentsPattern
+							}
+						};
+
 						if (options.compress) {
 
 							iOriginalSize = fileContent.length;
 							iPreloadOriginalSize += iOriginalSize;
 
+							// Override UglifyJS defaults with given options
+							if (typeof options.compress.uglifyjs === 'object') {
```

Comment:
I would rather go for the opposite direction and merge the defaults into the given object.
This will solve the `comments` issue mentioned in the other two comments.
Something like this (untested):
```js
// Convert default compression to empty configuration object
if (options.compress === true) {
  options.compress = {};
}

// Make sure to have an object
options.compress.uglifyjs = options.compress.uglifyjs || {};

// Always override given options, override shouldn't be possible
options.compress.uglifyjs.fromString = true;
options.compress.uglifyjs.warnings = grunt.option('verbose') === true;

// Set default "comments" option if not given already
options.compress.uglifyjs.output = options.compress.uglifyjs.output || {};
if (!options.compress.uglifyjs.output.hasOwnProperty("comments")) {
  options.compress.uglifyjs.output = copyrightCommentsPattern;
}
```